app/models/gibct_site_mapper.rb

```diff
@@ -32,8 +32,8 @@ def generate_sitemap(version)
     SitemapGenerator::Sitemap.create do
       add '/search', priority: 0.9, changefreq: 'monthly'
 
-      Institution.version(version).find_each(batch_size: Settings.active_record.batch_size.find_each) do |institution|
-        add "/profile/#{institution.facility_code}", priority: 0.8, changefreq: 'weekly'
+      Institution.version_id(version).find_each(batch_size: Settings.active_record.batch_size.find_each) do |i|
```

Comment:
why rename the variable?

Reply:
This was renamed to avoid another rubocop length instance